validator/models/validation_config.go

```diff
@@ -41,6 +42,9 @@ type validatorConfig struct {
 
 	MetricNamespace  string             `yaml:"metric_namespace"`
 	MetricValidation []MetricValidation `yaml:"metric_validation"`
+
+	CommitHash string `yaml:"commit_hash"`
+	CommitDate string `yaml:"commit_date"`
```

Comment:
Do we need to add these two fields to `test/stress/statsd/parameters.yml`?

Reply:
That's the fun thing regarding `unmarshall` . If you are nut suppling the variable (e.g `CommitHash`), it will give the null or "" depends on the yaml (e.g `mapstructure` return null while `yaml` return empty string).

Reply:
Yeah... agreed with the default value but will this cause an issue if `GetCommitInformation()` is called and `v.CommitDate` is null or ""
(Since it's calling `commitDate, _ := strconv.ParseInt(v.CommitDate, 10, 64)`)

Reply:
So there is no use case of integration test or stress test using `GetCommitInformation` and it seems to be overengineering in adding it.

Reply:
okk.. thanks for explaining.